client/src/commands/UserInputUtil.ts

```diff
@@ -163,18 +159,8 @@ export class UserInputUtil {
         return vscode.window.showQuickPick(options, quickPickOptions);
     }
 
-    public static async showPeerQuickPickBox(prompt: string, fabricConnection?: IFabricConnection): Promise<string | undefined> {
-        let connection: IFabricConnection;
-        if (!fabricConnection) {
-            const fabricConnectionManager: FabricConnectionManager = FabricConnectionManager.instance();
-            connection = fabricConnectionManager.getConnection();
-            if (!connection) {
-                throw new Error('No connection to a blockchain found');
-            }
-        } else {
-            connection = fabricConnection;
-        }
-
+    public static async showPeerQuickPickBox(prompt: string): Promise<string | undefined> {
+        const connection: IFabricConnection = await FabricRuntimeManager.instance().getConnection();
```

Comment:
is it possible that you could get here with the runtime stopped. i don't think `getConnection` starts the runtime?

Reply:
This is only called in installCommand.ts, and we check local_fabric is running before we call it.